### ISSUE TITLE
write rule for GNU-specific targets

### ISSUE THREAD

**Issue by meatballhat** [mrtazz/checkmake#15]
Some/all of these?  https://www.gnu.org/prep/standards/html_node/Standard-Targets.html#Standard-Targets

**Comment by mrtazz** [comment]
hmm I think it would be nice to have. However these will mostly matter in autotools generated Makefiles where they probably exist anyways. And there is no way someone will just implement all of them in a regular handwritten Makefile is my assumption. So yea let's add it but I don't think it's a priority :).

**Comment by meatballhat** [comment]
@mrtazz agreed not a priority.  I'm +0 to it.  Nice to have a doc to point at, but then again if this project gains enough steam then it will be the doc to point at 😉

**Comment by mrtazz** [comment]
Thinking about this again we could probably have this just as a config option for `minphony` if we make it take a proper config.

**Comment by cooljeanius** [comment]
Has dup #72 for the original issue reported here, and then bugs #86, #87, and PR #88 for this part:

> Thinking about this again we could probably have this just as a config option for [`minphony`](https://github.com/mrtazz/checkmake/blob/master/rules/minphony/minphony.go?rgh-link-date=2017-01-30T01%3A22%3A40Z) if we make it take a proper config.
